products/chef-supermarket.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Chef Supermarket
+category: server-app
+tags: ruby-runtime
+iconSlug: chef
+permalink: /chef-supermarket
+versionCommand: supermarket-ctl version
+releasePolicyLink: https://docs.chef.io/versions/
+changelogTemplate: "https://docs.chef.io/release_notes_supermarket/#__LATEST__"
+
+
+# https://docs.chef.io/versions/ only lists 5.x as supported. Historically Chef has had a roll-forward approach
```

Comment:
It would be nice to document this in the product description, along with details about the release policy, if there is one, and the support policy (for example, are release only supported with security fixes ?).

Reply:
I added this to the bottom of the description. I tried to keep it pretty neutral around the previous versions, but I worked on the team that built Supermarket in the past so I have some inside information. The SaaS Supermarket site is the same package and they've never backported a fix since the private Supermarket packages are an afterthought. Everything not 5.x is EOL.